components/brave_wallet/renderer/brave_wallet_js_handler.cc

```diff
@@ -0,0 +1,165 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/renderer/brave_wallet_js_handler.h"
+
+#include <utility>
+
+#include "base/no_destructor.h"
+#include "base/strings/utf_string_conversions.h"
+#include "brave/components/brave_wallet/resources/grit/brave_wallet_script_generated_map.h"
+#include "content/public/renderer/render_frame.h"
+#include "gin/arguments.h"
+#include "gin/function_template.h"
+#include "third_party/blink/public/common/browser_interface_broker_proxy.h"
+#include "third_party/blink/public/web/blink.h"
+#include "third_party/blink/public/web/web_local_frame.h"
+#include "third_party/blink/public/web/web_script_source.h"
+#include "ui/base/resource/resource_bundle.h"
+
+namespace {
+
+static base::NoDestructor<std::string> g_provider_script("");
```

Comment:
same as the other comment, why do we need a script for anything?

Reply:
removed script execution